### ISSUE TITLE
Importing aiohttp returns syntaxerror in python 3.7 win-32

### ISSUE THREAD

**Issue by PythonCoderAS** [aio-libs/aiohttp#3256]
## Long story short

I cannot use the aiohttp module.

## Expected behaviour

Aiohttp to be imported correctly
## Actual behaviour

I get a syntaxerror during import
## Steps to reproduce

Import aiohttp
```python

import aiohttp
```
## Your environment

**Client version**
Version: 1.0.5
OS: Windows 10

## Full traceback
```
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\IPython\core\interactiveshell.py", line 2961, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-3-3ac2d62558b8>", line 1, in <module>
    import aiohttp
  File "C:\Users\neela\AppData\Local\JetBrains\Toolbox\apps\PyCharm-C\ch-0\182.4129.34\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 20, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\aiohttp\__init__.py", line 10, in <module>
    from .protocol import *  # noqa
  File "C:\Users\neela\AppData\Local\JetBrains\Toolbox\apps\PyCharm-C\ch-0\182.4129.34\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 20, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\aiohttp\protocol.py", line 18, in <module>
    from .helpers import reify
  File "C:\Users\neela\AppData\Local\JetBrains\Toolbox\apps\PyCharm-C\ch-0\182.4129.34\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 20, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\aiohttp\helpers.py", line 25
    ensure_future = asyncio.async
                                ^
SyntaxError: invalid syntax
```
The first two lines are the IPython code to import stuff.

## The cause

In Python 3.7, asyncio.async is equivalent to typing async in async def, so the code is the equivalent to writing `ensure_future = def`. Python cannot finish the define, so it results in a syntax error. The only way to fix this is to make a separate package for Python 3.7 by omitting the code.

**Comment by Harmon758** [comment]
v1.0.5 was released almost two years ago, before Python 3.7 development even began.
This specific issue was fixed in v2.3.3 with https://github.com/aio-libs/aiohttp/commit/7e15124e78d3ec9e6752fbde51671480813d46ba almost a year ago.
Anything resembling this line was removed in v3.0.0 with https://github.com/aio-libs/aiohttp/commit/1a606bc1f08db2291a0d4dcd6a8b600e502ebc70 when Python 3.4 support was dropped.
Please make sure you're up to date with the latest development version when submitting issues.

Additionally, `async` is a keyword in Python 3.7. That's what's causing the syntax error. This does not mean `async` is equivalent to `async def` or `def`.
> The only way to fix this is to make a separate package for Python 3.7 by omitting the code.

This was also not true, as seen by the v2.3.3 fix.

**Comment by asvetlov** [comment]
Aiohttp 1.x version is not supported anymore
